Source/Core/DolphinQt2/GameList/GameTracker.cpp

```diff
@@ -21,6 +23,11 @@ GameTracker::GameTracker(QObject* parent) : QFileSystemWatcher(parent)
   connect(this, &QFileSystemWatcher::directoryChanged, this, &GameTracker::UpdateDirectory);
   connect(this, &QFileSystemWatcher::fileChanged, this, &GameTracker::UpdateFile);
 
+  cache.Load();
+
+  for (const auto& file : cache.GetCached())
```

Comment:
Calling `addPath` on the cached paths will update them if they change while Dolphin is running, but won't update them on startup. If they're in the configured paths, then this is redundant; if they're not in the configured paths, then shouldn't be added anyways. So, this logic should be removed.